src/core/Microsoft.PowerToys.Settings.UI.Lib/KeyboadManagerConfigModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.PowerToys.Settings.UI.Lib
+{
+    public class KeyboadManagerConfigModel
+    {
+        [JsonPropertyName("remapKeys")]
+        public List<KeysDataModel> RemapKeys { get; set; }

```

Comment:
@arjunbalgovind with your second scenario, where they switch between these. they would both be on a single config file? or that they switch to a config file with in-process remapping

Reply:
For both scenarios I meant single config file. A third scenario would be where you would need to switch to another config for that. Or in essence your config file only stores one method of key to key remapping.

Reply:
I have included the `inProcess` subkey in the data model for now. @saahmedm for the case if we want both the approaches independent what will happen when user choose to select the regsitry approach will we move all the single key remappings to registry(if possible) or have a separate list of remapping for both of the cases?

Reply:
missed this guys. imo at one point users should be able to remap keys in-process and registry level at the same time. so the first scenario @arjunbalgovind mentioned.

Reply:
I think we can make that change on a separate PR, since this already merged. @saahmedm can you make an issue to track adjusting the JSON to support future in-process and registry remaps? We can mark that as a P0 so that the change is made before Build.